lib/util/fs/index.ts

```diff
@@ -300,3 +300,8 @@ export async function getLocalFiles(
 
   return fileContentMap;
 }
+
+const validFilePathRegex = new RegExp(/^(\/?[a-z0-9.]+)+$/);
+export function isValidPath(s: string): boolean {
```

Comment:
```suggestion
export function isValidLocalPath(s: string): boolean {
```
disallow absolute paths, so no leading slash.

we should also disallow `./../` to not going outside of the repo.

i think we already have a function to validate that a path is inside the repo .